tools/template_based_query_generation/src/main/java/data/Column.java

```diff
@@ -0,0 +1,122 @@
+package data;
+
+import parser.Utils;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+
+public class Column {
```

Comment:
Deleted the column class and moved the data generation methods to Table. Now the data is store in ArrayList<ArrayList<?>> instead of ArrayList<Column>